Backend/DataAccessLayer/Repositories/PointRepository.cs

```diff
@@ -0,0 +1,71 @@
+﻿using DataAccessLayer.Database;
+using DataAccessLayer.Models;
+using System;
+using System.Data.Entity;
+using System.Linq;
+
+namespace DataAccessLayer.Repositories
+{
+    public class PointRepository
+    {
+        private Point CheckPointLimits(Point point)
+        {
+            if (point.Longitude > 180 || point.Longitude < -180 ||
+                point.Latitude > 90 || point.Latitude < -90)
+                return null;
+            return point;
+        }
+
+        public bool ExistingPoint(DatabaseContext _db, Point point)

```

Comment:
Not sure what the point of this function is. You already have a point object - are you checking to see if that point is persisted in the database already?

Reply:
Yes, the same function exists for user management, so I just kept the naming convention

Reply:
Hmm, don't think it's that useful then. It really doesn't validate much.

Reply:
I'll just delete this function then.